### PR TITLE
Maya: make render settings validator optional

### PR DESCRIPTION
**Pull request by antirotor**
## Changelog Description
Make Render Settings validator otional

## Testing notes:
1. set render settings validator in Maya optional
2. try to publish render

### CODE REVIEW

**Comment by moonyuet** [comment]
I also find out the enabled option for `ValidatePluginPathAttributesModel` might be wrong.(not related to this PR)
https://github.com/ynput/ayon-core/blob/9f4c7018a289a2bd060bd8ad1c25969952e93c28/server_addon/maya/server/settings/publishers.py#L239-L246

File: server_addon/maya/server/settings/publishers.py [hunk 1]
```diff
@@ -265,6 +265,7 @@ class RendererAttributesModel(BaseSettingsModel):
 
 
 class ValidateRenderSettingsModel(BaseSettingsModel):
+    optional: bool = SettingsField(title="Optional")
```
[the remark this answers is not in the document]
Comment (BigRoy) [reply server_addon/maya/server/settings/publishers.py:268]: While we're at it - should we also expose `active`?

**Comment by antirotor** [comment]
> I also find out the enabled option for `ValidatePluginPathAttributesModel` might be wrong.(not related to this PR)

Unrelated, but fixed it here anyway, sorry about that.

File: server_addon/maya/server/settings/publishers.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (BigRoy) [reply server_addon/maya/server/settings/publishers.py:269]: Jut to double check - this `active` toggle wasn't implemented yet? Correct?

**Review by moonyuet — APPROVED** [review]
Tested successfully in Maya, one remark is that it would be great to have more concise label for the validator such as `Validate Render Setting` so that users know what the validator is enabled for the render instance when they are at the publish tab. 
https://github.com/ynput/ayon-core/blob/b87e9049ca9577afb9e857c0ecc49fbe93023cc8/client/ayon_core/hosts/maya/plugins/publish/validate_rendersettings.py#L72
image

**Review by LiborBatek — APPROVED** [review]
All works as expected...

I have located the ```Validator``` and set it as ```Optional``` and it was available in the publisher UI and when my settings been wrong it correctly validated when enabled and vice versa....
Screenshot 2024-05-02 152503


Screenshot 2024-05-02 152828

Screenshot 2024-05-02 152854


Screenshot 2024-05-02 153044

**Review by BigRoy — CHANGES_REQUESTED** [review]

File: server_addon/maya/server/settings/publishers.py [hunk 2]
```diff
@@ -1171,6 +1174,8 @@ class PublishersModel(BaseSettingsModel):
         ]
     },
     "ValidateRenderSettings": {
+        "enabled": True,
+        "optional": False,
```
Comment (BigRoy) [inline server_addon/maya/server/settings/publishers.py:1178]: This is missing the default value for the `active` setting @antirotor - you may have merged it too quickly?

Comment (antirotor) [reply server_addon/maya/server/settings/publishers.py:1178]: damn, the default value

Comment (antirotor) [reply server_addon/maya/server/settings/publishers.py:1178]: I've added it in #481

**Pull request merged**